app/src/main/java/ch/epfl/skysync/navigation/MainGraph.kt

```diff
@@ -18,7 +18,7 @@ fun MainGraph(
 ) {
   NavHost(
       navController = navHostController,
-      startDestination = if (user == null) Route.LOGIN else Route.MAIN) {
+      startDestination = /*if (user == null) Route.LOGIN else */ Route.MAIN) {
```

Comment:
Is the null check for the user performed elsewhere ?

Reply:
No, it was just to bypass when I tested it manually and I forgot to put it back